spec/toplevelmenu/checklist.txt

```diff
@@ -0,0 +1,59 @@
+Description block & feature	Result
+
+TopLevelMenu	////////////
+"Verify that user can hover on the available
+tabs on the TopLevelMenu"	Pass
+"Verify that user can click on the available
+tabs on the TopLevelMenu"	Pass
+"Verify that user can click on the subtabs 
+on the Women tab"	Pass
+"Verify that user can click on the subtabs 
+on the Dresses tab"	Pass
```

Comment:
Verify that user can click on the subtabs  on the Dresses tab
Means that we test if user has ability to click
Verify upon clicking on Dresses tab user is redirected to Dresses page.
Applies to all such checklists